courses/factories.py

```diff
@@ -19,6 +31,17 @@ class Meta:  # pylint: disable=missing-docstring
 class ModuleFactory(DjangoModelFactory):
     """Factory for Module"""
     course = factory.SubFactory(CourseFactory)
+    title = fuzzy.FuzzyText(prefix="Module ")
 
     class Meta:  # pylint: disable=missing-docstring
         model = Module
+
+    @factory.lazy_attribute
+    def subchapters(self):  # pylint: disable=no-self-use
+        """
+        Generate subchapter listing.
+        """
+        rand = random.Random(fuzzy.get_random_state())
```

Comment:
Note to reviewer, FactoryBoy wants to manage randomness so it can generate repeatably failing test results.

http://factoryboy.readthedocs.org/en/latest/fuzzy.html#managing-randomness